package.json

```
@@ -7,19 +7,18 @@
   ],
   "packageManager": "yarn@1.22.19",
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.60.1",
-    "@typescript-eslint/parser": "^5.60.1",
-    "eslint": "^8.43.0",
+    "@eslint/js": "^9.2.0",
+    "@typescript-eslint/eslint-plugin": "^7.8.0",
+    "@typescript-eslint/parser": "^7.8.0",
+    "eslint": "^9.2.0",
     "eslint-config-prettier": "^8.8.0",
     "prettier": "^2.8.8",
-    "tslint": "^6.1.3",
-    "tslint-config-prettier": "^1.18.0",
-    "tslint-plugin-prettier": "^2.3.0",
-    "typescript": "^5.4.5"
+    "typescript": "^5.4.5",
+    "typescript-eslint": "^7.8.0"
   },
   "scripts": {
-    "lint": "tslint 'packages/*/src/**/*.ts'",
-    "eslint": "eslint 'packages/*/src/**/*.ts'",
+    "lint": "eslint 'packages/*/src/**/*.ts'",
```

What do you think of maybe passing `--quiet` here? That will hide `warn` level rule violations. That means we'll still see warnings in our editors (like explicit `any`), but when we get a lint failure, we'll just see the errors?

```suggestion
    "lint": "eslint 'packages/*/src/**/*.ts' --quiet",
```

Docs: https://eslint.org/docs/latest/use/command-line-interface#--quiet